microsoft/testsuites/network/networksettings.py

```diff
@@ -192,25 +200,39 @@ def validate_device_enabled_features(self, node: Node) -> None:
             into affect.
 
             Steps:
-            1. Get the device's generic-receive-offload and large-receive-offload
+            1. Get all the device's generic-receive-offload and large-receive-offload
                 settings.
-            2. Try flipping the GRO and LRO settings and validate it takes affect.
-            3. Revert back the settings to original values.
+            2. If both GRO and LRO settings are "[fixed]" then skip testing specific
+                device.
+            3. Try flipping the GRO and LRO settings and validate it takes affect.
+            4. Revert back the settings to original values.
         """,
         priority=1,
     )
     def validate_device_gro_lro_settings_change(self, node: Node) -> None:
         ethtool = node.tools[Ethtool]
 
+        skip_test = True
         devices_gro_lro_settings = ethtool.get_all_device_gro_lro_settings()
+        for settings in devices_gro_lro_settings:
+            interface = settings.interface
+            if settings.gro_fixed and settings.lro_fixed:
+                self.__log.info(
```

Comment:
Sorry, not find it at first time. Please refer to latest code to get the log object. It should be from method arguments, not test suite. So that the log can be visible in the test case level log.